src/main/java/uk/ac/ebi/ampt2d/accession/AccessioningObject.java

```diff
@@ -23,4 +23,8 @@
 
     T getAccession();
 
+    void setHash(String hash);
+
+    void setAccession(T accession);
+
 }
```

Comment:
I still think it would be more descriptive if this interface was called `AccessionedObject` or `IAccessionedObject`, depending on whether there is an abstract class implementing it or not. See also [previous comment](https://github.com/EBIvariation/amp-t2d-accession/pull/6#discussion_r161199422).